ols/app/models/config.py

```diff
@@ -211,22 +256,18 @@ def __init__(self, data: Optional[dict] = None):
         super().__init__()
         if data is None:
             return
-        self.llm_config = LLMConfig(data.get("llm_providers", None))
-        self.ols_config = OLSConfig(data.get("ols_config", None))
+        v = data.get("LLMProviders")
+        if v is not None:
+            self.llm_config = LLMConfig(v)
```

Comment:
1. Shall we keep the key using underscore as delimiter as other models do, for a better consistency? (`llm_providers` vs `LLMProviders`)
2. I think it more readable in this way:
```python
if "LLMProviders" in data:
    self.llm_config = LLMConfig(data["LLMProviders"])
```

Reply:
> Shall we keep the key using underscore as delimiter as other models do, for a better consistency?

yes, when i wrote the initial example config.yaml i went with python-style naming, i think we should stick with that.  So "llm_providers" and not "LLMProviders", as the field name in the yaml.

Reply:
I'm leaning towards renaming the LLMConfig class into LLMProviders, the Config.llm_config attribute into Config.llm_providers.

Reply:
@raptorsun 
```
if "LLMProviders" in data:
    self.llm_config = LLMConfig(data["LLMProviders"])
```
is not entirely equivalent, if there's a dict entry "LLMProviders"->None, it'll pass that None on.

Reply:
yes, let's keep it that way to prevent the case where "LLMProviders" entry has None value.